Models/EmployeeSalesmodel.cs

```diff
@@ -0,0 +1,18 @@
+public class EmployeeSalesModel
+{
+    // Campos de SalesModel
+    public int SalesOrderID { get; set; }
+    public DateTime OrderDate { get; set; }
+    public DateTime DueDate { get; set; }
+    public DateTime ShipDate { get; set; }
+    public string? Status { get; set; }
+    public int SalesOrderDetailID { get; set; }
+    public int CustomerID { get; set; }
+    public decimal SalesYTD { get; set; }
+    public decimal SalesLastYear { get; set; }
+
+    // Campos de PersonModel
+    public string? FirstName { get; set; }
```

Comment:
If you've determined that there's a relationship with another table in this model, instead of declaring the fields as properties, you can use the entire object or declare a new object of the PersonModel class. It's true that if you don't need all the data, you can just use the methods you need in the parseMethod. Here's an example: [link](https://www.codeproject.com/Questions/5248683/How-I-can-use-the-same-model-for-another-view-with)